src/main/java/org/isf/admission/gui/AdmissionBrowser.java

```diff
@@ -2260,7 +2261,7 @@ public void actionPerformed(ActionEvent e) {
 					}
 
 					if (!result) {
-						JOptionPane.showMessageDialog(AdmissionBrowser.this, MessageBundle.getMessage("angal.sql.thedatacouldnotbesaved"));
+						MessageDialog.error(null, "angal.common.data.not.saved.msg");
```

Comment:
the "dot" notation in bundles is useful to group subkeys while I don't see the benefit to use it as words separator

Reply:
I found that I couldn't understand some of the long run on key values; this made it clearer for my simple mind.

Reply:
I have to the best of my ability removed the "dot" notation.

Reply:
Thanks!